src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -100,7 +101,7 @@ public static void checkUser(Request req) {
      * @return true if the request can operate without a user profile, false otherwise.
      */
     private static boolean expectsMissingProfile(Request req, RequestingUser profile) {
```

Comment:
The comment on this method should probably be updated and perhaps also the log message on line 78.

Reply:
Thanks, updated in https://github.com/ibi-group/otp-middleware/pull/213/commits/d73eb97328616713dbe7d034f4f3e926434ab291.